### DISCUSSION TITLE
Enable sharing Syntax

### DISCUSSION THREAD

**Discussion by coacharnold1** [navidrome/navidrome#2789] [Q&A]
Hi Folks ...   if someone could just share their Docker-compose file for navidrome I could solve this fast.  How should sharing be enabled in Docker-Compose .   

What I have isn't working:

ND_ENABLESHARING: true

is this correct or does in need quotes, as in:

ND_ENABLESHARING: "true"

**Comment by jsaathof** [comment]
Are you using the environment option in the docker-compose file? I believe that the correct usage is:

`ND_ENABLESHARING=true`

See https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-environment-attribute

**Comment by coacharnold1** [comment]
Thanks JSAATOF this was ridiculous to sort out ..... hopefull this will help someone .... here's what I have now that works as my yml file.... hopefulle this helps someone else ...  .... 

Version: "3"
services:
  navidrome:
    image: deluan/navidrome:latest
    user: 1000:1000 # should be owner of volumes
    ports:
      - "4533:4533"
    restart: unless-stopped
    environment:
      # Optional: put your config options customization here. Examples:
      ND_UIWELCOMEMESSAGE: "Welcome to My Audio Server - Don't Go Crazy"
      ND_SCANSCHEDULE: "30 3 * * *"
    #  ND_SCANSCHEDULE: 1m
      ND_LOGLEVEL: info
      ND_SESSIONTIMEOUT: 24h
      ND_LISTENBRAINZ_ENABLED: "false"
      ND_IGNOREDARTICLES: "The El La Los Las Le Les Os As O A"
      ND_ENABLESHARING: "true"
      ND_ENABLESTARRATING: "true"
      ND_ENABLEFAVOURITES: "true"
      ND_COVERARTPRIORITY: "cover.*, folder.*, front.*, embedded, external"
      ND_LASTFM_APIKEY: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
      ND_LASTFM_SECRET: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"

    volumes:
      - "/home/computer/navidrome/data:/data"
      - "/home/computer/media-bind:/music:ro"
